lib/ansible/modules/network/netvisor/pn_prefix_list.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+# Copyright: (c) 2018, Pluribus Networks
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = """
+---
+module: pn_prefix_list
+author: "Pluribus Networks (@rajaspachipulusu17)"
+version_added: "2.9"
+short_description: CLI command to create/delete prefix-list
+description:
+  - This module can be used to create or delete prefix list.
+options:
+  pn_cliswitch:
+    description:
+      - Target switch to run the CLI on.
+    required: false
+    type: str
+  state:
```

Comment:
`state` parameter shouldn't be a required param, you can set the default value for `state` as `present` and make any other module parameter as required param (e.g. pn_cliswitch/pn_name)

Reply:
@justjais Its a standard what we have used for all modules. If it is mandatory and required i can changed it else i would prefer to use same.

Reply:
@rajaspachipulusu17, apologies if it was missed in earlier review but to be consistent with Ansible I had suggested this change, so if from this module you can start implementing the suggestion I made would be great, coz if you think user can run the module just by passing `state` as parameter which ultimately shall do nothing. So ideally the parameter that's mandatory from module functionality prospects should be made `required=True`.
ref: I can run the module as following, which will do nothing:
```
- name: Create prefix list
  pn_prefix_list:
    state: "present"
```